dodiscover/replearning/gin.py

```diff
@@ -0,0 +1,116 @@
+"""
+Wrapper for the GIN latent variable causal discovery algorithm in causal-learn
+"""
+
+from pandas import DataFrame
+
+from causallearn.search.HiddenCausal.GIN.GIN import GIN as GIN_
+from pywhy_graphs import CPDAG
+
+
+class GIN:
+    """Wrapper for GIN in the causal-learn package.
```

Comment:
Probably also add reference? 

See other classes in constraint-based algo classes, for how I auto-linked bib files. You'll need to add a bibtex item to the `doc/references.bib` file (keeping it in alphabetical order if possible) and then reference it here and add a `References` section.

Reply:
[e9fb4ca](https://github.com/py-why/dodiscover/pull/72/commits/e9fb4ca2c41b0d5aad2020c1afdedc796d352a2c)